### ISSUE TITLE
Inconsistent behaviour of arange

### ISSUE THREAD

**Issue by sscardapane** [google/jax#145]
I have installed JAX on Colab as in the QuickStart notebook. Then I run this instruction:

```
import jax.numpy as np
print(np.arange(0.0, 1.0, 0.1))
```

And I get an array with a single value [0], which is different from:

```
import numpy as np
print(np.arange(0.0, 1.0, 0.1))
```

Where I (correctly) get an array with 11 values 0, 0.1, ..., 0.9, 1.0. Is this intended?

**Comment by mattjj** [comment]
Not intended, that's a bug! A temporary workaround is to `import numpy as onp` and just use `onp.arange` here.

**Comment by sscardapane** [comment]
Ok, thanks a lot! Love the library!

**Comment by mattjj** [comment]
Thanks for the kind words!

I pushed a fix in 6a9952a and will merge it in when the CI tests pass (in ~15 minutes). I'll also update the pypi package with the fix.

We really appreciate you spotting this!
